### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by jepunband** [iojw/socialscan#60]
hi just cloned and tried to use the tool today. It does not work.

----------------------------------------          
               jasonlkh
----------------------------------------
Lastfm
GitHub
GitLab
Reddit
Tumblr
Twitter
Yahoo
Instagram: QueryError - Could not retrieve token. You might be sending too many requests. Use a proxy or wait before trying again.
Snapchat: KeyError - 'reference'

**Comment by sinawic** [comment]
i get kinda the same in v2.0.0

```
Instagram: QueryError - Could not retrieve token. You might be sending too many requests. Use a proxy or wait before trying again.
Lastfm: QueryError - Unexpected content type text/html. You might be sending too many requests. Use a proxy or wait before trying again.
Snapchat: KeyError - 'Set-Cookie'
Yahoo: KeyError - 'errors'
```

**Comment by Dave181295** [comment]
Whats the solution?

**Comment by iojw** [comment]
This should be fixed by @harveyrandall contribution published in 2.0.1. Thank you for your patience!
